### PR TITLE
US 10: Create Intern Model

### PR DESCRIPTION
**Pull request by lcruz94**
US 10: Create Intern Model

### CODE REVIEW

**Comment by lcruz94** [comment]
US 10: Create Intern Model

**Review by cdelacruz8512 — COMMENTED** [review]
Change from "InternrModel" to "Intern"
TechnologiesSummary need to be into another class file

**Review by cdelacruz8512 — COMMENTED** [review]
Change from fourthSourceOrg to FourthSourceOrg

**Review by cdelacruz8512 — APPROVED** [review]
The code is correct

File: Internship.API/Models/Intern.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+using System;
+
+namespace Intern.Models
+{
+    public class Intern
+    {
+        [BsonId]
+        [BsonRepresentation(BsonType.ObjectId)]
+
+        [BsonElement("Intern")]
```
Comment (fasaga) [inline Internship.API/Models/Intern.cs]: Can you remove line 12? It is no needed

File: Internship.API/Models/TechnologiesSummary.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+using MongoDB.Bson.Serialization.Attributes;
+using System;
+
+namespace TechnologiesSummaryModel.Models
+{
+    public class TechnologiesSummary
+    {
+        public string starting { get; set; }
+        public string final { get; set; }
+
```
Comment (fasaga) [inline Internship.API/Models/TechnologiesSummary.cs]: Add anew prop called "all" type of array of string "string[]"

File: Internship.API/Models/TechnologiesSummary.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using System;
+
+namespace TechnologiesSummaryModel.Models
+{
+    public class TechnologiesSummary
+    {
+        public string starting { get; set; }
```
Comment (fasaga) [inline Internship.API/Models/TechnologiesSummary.cs]: for these props can you use the Standard to name them?

File: Internship.API/Models/Intern.cs [hunk 4]
```diff
@@ -0,0 +1,29 @@
+﻿using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using System;
+
+namespace Intern.Models
```
Comment (fasaga) [inline Internship.API/Models/Intern.cs]: change namespace to Intership.API.Models

File: Internship.API/Models/TechnologiesSummary.cs [hunk 5]
```diff
@@ -0,0 +1,13 @@
+﻿using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using System;
+
+namespace TechnologiesSummaryModel.Models
```
Comment (fasaga) [inline Internship.API/Models/TechnologiesSummary.cs]: change namespace to Intership.API.Models

File: Internship.API/Models/Intern.cs [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+{
+    public class Intern
+    {
+        [BsonId]
+        [BsonRepresentation(BsonType.ObjectId)]
+        [Required]
+        public int UserId { get; set; }
+        [Required]
+        public int MentorId { get; set; }
+        public string TechnologiesSummary { get; set; }
```
Comment (fasaga) [inline Internship.API/Models/Intern.cs]: This property is not string type, it should be "TechnologiesSummary" instead of string

**Pull request merged**